.github/workflows/release_dev.yml

```
@@ -10,16 +10,41 @@
+
+env:
+  MAIN_TAG: nightly
+  PREV_TAG: nightly-prev
 
 jobs:
+  release_type:
+    name: Release Type
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)